satpy/composites/__init__.py

```diff
@@ -1666,12 +1666,10 @@ def __call__(self, *args, **kwargs):
         img.attrs.pop("modifiers", None)
         img.attrs.pop("calibration", None)
         # Add start time if not present in the filename
-        if "start_time" not in img.attrs or not img.attrs["start_time"]:
-            import datetime as dt
-            img.attrs["start_time"] = dt.datetime.utcnow()
-        if "end_time" not in img.attrs or not img.attrs["end_time"]:
-            import datetime as dt
-            img.attrs["end_time"] = dt.datetime.utcnow()
+        if "start_time" not in img.attrs:
+            img.attrs["start_time"] = None
+        if "end_time" not in img.attrs:
+            img.attrs["end_time"] = None
```

Comment:
Can you remind me, what does the `generic_image` reader do if it can't find start/end time from the filename? None or current time?

I guess if `None` then this isn't needed, right?

Reply:
Right you are, seems to be `None`. I'll adjust when back on the work computer.

Reply:
The `StaticImageCompositor` tests mock the image reading, so the times are not set. I'll remove these assertions.